build_tools/clientSideJasmine.js

```diff
@@ -17,16 +16,11 @@ function clientSideJasmine(config) {
     );
 
     return gulp.src(files)
-        .pipe(plumber({
-            errorHandler: function() {
-                process.exit(1);
-            }
-        }))
         .pipe(jasmineBrowser.specRunner({
             console: !debug
         }))
-        .pipe(gulpif(!debug, jasmineBrowser.headless()))
-        .pipe(gulpif(debug, jasmineBrowser.server()));
+        .pipe(gulpif(debug, jasmineBrowser.server()))
+        .pipe(gulpif(!debug, jasmineBrowser.headless()));
```

Comment:
turns out it's just about the order of these 2 pipes: https://github.com/jasmine/gulp-jasmine-browser/issues/20#issuecomment-167723754